terraform/modules/emblem-app/iam.tf

```
@@ -19,6 +19,12 @@ resource "google_service_account" "api_manager" {
   display_name = "api-manager"
 }
 
+resource "google_project_iam_member" "project" {
```

Suggestion: use a more descriptive resource name
```suggestion
resource "google_project_iam_member" "firestore_user" {